components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -41,12 +41,12 @@ const char kObservingScriptletEntryPoint[] =
 
 const char kScriptletInitScript[] =
     R"((function() {
-          let text = %s;
+          let text = 'let deAmpEnabled = %s;\n' + %s;
```

Comment:
This is a bit ugly, but we can't just add `let deAmpEnabled = %s;` before this line because `deAmpEnabled` will then not be visible to the scriptlet when the script is actually executed as part of `appendChild` on line 50.

Reply:
do we need
```js
let text = '(function() {\nlet deAmpEnabled = %s;\n' + %s + '})()';
```
?

Reply:
In the current form, `deAmpEnabled` just becomes part of the scriplet and will go out of scope once the scriptlet is done executing, right? Which is what we want.

Reply:
The actual scriptlet is that second `%s` above. `text` becomes essentially `<script>text</script>`, so I think the `deAmpEnabled` needs its own IIF scope as well.

Reply:
In other words, the code you have right now will currently inject something like this:

```html
<script>
let deAmpEnabled = true;
try {
  (function() {
    // scriptlet 1
  })()
} catch (e) {}
try {
  (function() {
    // scriptlet 2
  })()
} catch (e) {}
</script>
```

Reply:
Ah that's right, good catch. Updated